packages/block-editor/src/components/plain-text/index.native.js

```diff
@@ -16,22 +16,39 @@ import styles from './style.scss';
 export default class PlainText extends Component {
 	constructor() {
 		super( ...arguments );
-		this.isIOS = Platform.OS === 'ios';
+		this.isAndroid = Platform.OS === 'android';
 	}
 
 	componentDidMount() {
 		// if isSelected is true, we should request the focus on this TextInput
 		if ( this._input.isFocused() === false && this.props.isSelected ) {
-			this.focus();
+			if ( this.isAndroid ) {
+				/*
+				 * There seems to be an issue in React Native where the keyboard doesn't show if called shortly after rendering.
+				 * As a common work around this delay is used.
+				 * https://github.com/facebook/react-native/issues/19366#issuecomment-400603928
+				 */
+				this.timeoutID = setTimeout( () => {
+					this._input.focus();
+				}, 100 );
```

Comment:
Android TextInput has a bug where the keyboard doesn't pop when autoselecting a view in componentDidMount. to get around this a small timeout is set. This is an issue with ReactNative that seems to be opened then closed multiple times due to inactivity.

Reply:
Actually I think that this is a core Android bug where you can't focus an EditText if the view isn't active, so you need to wait with a timeout 😄